src/depthcloud_encoder.cpp

```diff
@@ -127,6 +133,22 @@ void DepthCloudEncoder::connectCb()
   }
 }
 
+void DepthCloudEncoder::cameraInfoCb(const sensor_msgs::CameraInfoConstPtr& cam_info_msg)
+{
+  if (cam_info_msg) {
+    // Update focal length value.
+    const double fx = f_mult_factor_ * cam_info_msg->K[0];
+    const double fy = f_mult_factor_ * cam_info_msg->K[4];
+    f_ = (fx + fx) / 2;
```

Comment:
`fx + fy`?

I was actually wondering how is a single focal value computed from `fx` and `fy`.